src/Accounts/CreateAccount/CreateAccount.js

```diff
@@ -44,7 +54,7 @@ class CreateAccount extends Component {
 
     return (
       <div>
-        <nav className='header-nav -modal'>
+        <nav className='header-nav'>
```

Comment:
At one point we should try factorizing this header component, giving it props like `title`, `leftComponent` and `rightComponent`